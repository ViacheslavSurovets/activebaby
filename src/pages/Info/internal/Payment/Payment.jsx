import React from 'react';
import { Paragraph } from '@commonStyles';

const Payment = ( props ) => (
  <>
    <Paragraph>Оплата производится в белорусских рублях в кассе магазина, наличным и безналичным платежом. Принимаются карты Visa, Mastercard, БелКард </Paragraph>
    <Paragraph>Произодится расчет картой рассрочки Халва</Paragraph>
    <Paragraph>Также можно перечислить средства на расчетный счет нашей организации.</Paragraph>
    <Paragraph>Поможем с доставкой по Беларуси, по городу Гродно доставим бесплатно.</Paragraph>
  </>
);

export default Payment;
