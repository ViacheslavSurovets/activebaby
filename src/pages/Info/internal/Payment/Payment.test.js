import React from 'react';
import { shallow } from 'enzyme';
import Payment from './Payment';


describe ( 'Payment', () => {

  it ( 'Payment Snapshot', () => {
    expect ( shallow ( <Payment  /> ) ).toMatchSnapshot ();
  } );
} );
