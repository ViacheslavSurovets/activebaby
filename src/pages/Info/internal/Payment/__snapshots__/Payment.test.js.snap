// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Payment Payment Snapshot 1`] = `ShallowWrapper {}`;
